DataFormats/GeometryVector/src/TrackingJacobians.cc

```diff
@@ -36,10 +36,8 @@ AlgebraicMatrix65 jacobianCurvilinearToCartesian(const GlobalVector& momentum, i
   R(5, 5) = 1.;
 
   double p = pvec.mag(), p2 = p * p;
```

Comment:
p2 = pvec.mag2(), overp = 1./sqrt(p2);
may be more precise and avoids one division below;
one may compute overpt to avoid one more division....